plone-6.0.x.cfg

```diff
@@ -11,4 +11,10 @@ zodb-temporary-storage = off
 plone.rest =
 black = 23.3.0
 pygments = 2.14.0
-grpcio-tools = 1.59.0
\ No newline at end of file
+
+# all this is neccessary to make the p.a.contenttypes (robotframework) test dependency happy :(
```

Comment:
Should these pins be updated in buildout.coredev then?

Reply:
Philip already took care of this in his PR: https://github.com/plone/buildout.coredev/pull/890

I just copied over those version pins...

Reply:
Okay, so probably they are only needed temporarily until that PR is merged. 👍